CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG for ASAB WebUI Components
 
+## 25.2.5
```

Comment:
@aringocode The version should be updated in `package.json` as well

Reply:
@Pe5h4 updated